src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -420,7 +424,25 @@ export default defineComponent({
         this.publishedText = this.data.publishedText
       }
 
-      if (typeof (this.data.publishedText) !== 'undefined' && this.data.publishedText !== null && !this.isLive) {
+      if (this.data.isRSS && this.data.publishedDate != null && !this.isLive) {
+        const now = new Date()
+        // Convert from ms to second to minute
+        let timeDiffFromNow = ((now - this.data.publishedDate) / 1000) / 60
+        let timeUnit = 'minute'
+
+        if (timeDiffFromNow > 60) {
+          timeDiffFromNow /= 60
+          timeUnit = 'hour'
+        }
+
+        if (timeUnit === 'hour' && timeDiffFromNow > 24) {
+          timeDiffFromNow /= 24
+          timeUnit = 'day'
+        }
```

Comment:
Could you please add support for seconds (it's rare but happens) as well as months and years.

I'm subscribed to one channel that hasn't uploaded in 3 years but i'm subscribed to keep it easily accessible.